polyply/src/gmx_energy_minize.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import subprocess
+from pathlib import Path
+import tempfile
+import vermouth
+from .dispatch_gmx import DispatchGMX
+
+
+def _find_max_force(mdrun_out):
+    for line in mdrun_out.split("\n"):
+        if "Maximum force" in line:
+            force = line.split()[3]
+    return float(force)
+
+class GMXMinimize():
```

Comment:
Should this not be a subclass of GMXDispatch? Or am I confused about what that should do?